packages/core/admin/webpack.config.dev.js

```diff
@@ -43,6 +43,28 @@ module.exports = () => {
 
   return {
     ...config,
+    snapshot: {
+      managedPaths: [
+        path.resolve(__dirname, '../core/@strapi/plugin/content-manager'),
```

Comment:
No we also need the CTB, upload because they are loaded the same way as the CM and the helper plugin because we need a full cache invalidation when we rebuild it.

Reply:
Yes I just meant the path /@strapi/plugin/xxx doesn't exist at all on disk it's juste core/xxxx for CM / CTB and upload